### ISSUE TITLE
Warning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead

### ISSUE THREAD

**Issue by demdecuong** [lipiji/TranSummar#13]
Warning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead. (masked_fill__cuda at /pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19)
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.

I recieve this warning, so how can i avoid this warning during training.

My environment:
- Pytorch version :1.5.1
- nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2019 NVIDIA Corporation
Built on Wed_Apr_24_19:10:27_PDT_2019
Cuda compilation tools, release 10.1, V10.1.168

**Comment by lipiji** [comment]
@demdecuong Thanks. This ia because some old pytorch version degraded funcs are used in my code. This issue has been Fixed.
